tests/conftest.py

```diff
@@ -236,3 +240,98 @@ def x12_with_custom_delimiters() -> str:
             "IEA|1|000000907?",
         ]
     )
+
+
+@pytest.fixture
+def x12_271_control_header() -> str:
+    return "\n".join(
+        [
+            "ISA*03*9876543210*01*9876543210*30*000000005      *30*12345          *131031*1147*^*00501*000000907*1*T*:~",
+            "GS*HB*000000005*54321*20131031*1147*1*X*005010X279A1~\n",
+        ]
+    )
```

Comment:
The 271 header looks the same as the 270 header.  Can these be consolidated as well?

Reply:
The difference in the headers are in the `GS` segments, `GS01`.
In the 270 `GS01 == HS` while in the 271 `GS01 == HB`.

One thought is we could refactor the conftest.py a bit and utilize "general" header and footer fixtures such as x12_control_header and x12_control_footer. We could then just update these fixtures in the larger "input" fixture such as https://github.com/dixonwhitmire/x12/blob/271-implementation/tests/conftest.py#L332.

How does that sound?

Reply:
I missed the 1-char difference!  I wouldn't do any further refactoring, given that.

Reply:
I did go ahead with the refactoring.  I think it does clean things up a bit. I suppose it's a trade-off between adding additional fixtures, parameterizing the fixtures, or just updating the fixtures in the other fixtures. 

The refactor isn't weighing down the tests. We can still execute all 91 tests in under a second locally.